apps/browser/qml/pages/PrivacySettingsPage.qml

```diff
@@ -50,6 +51,49 @@ Page {
                     description: qsTrId("settings_browser-la-clear_history_description")
                 }
 
+                ComboBox {
+                     id: historyErasingComboBox
+
+                     width: parent.width
+                     //% "Clear browser history for"
+                     label: qsTrId("settings_browser-la-clear_history_period")
+                     value: {
+                         switch (currentIndex) {
+                         case 0:
```

Comment:
could you put it to something like this
`
var historyPeriods {
 0 : HistoryModel.Everything,
 1 : HistoryModel.Day,
 ....
}
`